install_pkg.sh

```
@@ -0,0 +1,91 @@
+DEST_LOCAL="$GH_SHARE_LOCAL/$PKG_NAME"
+DEST_REMOTE="$GH_SHARE_REMOTE/$PKG_NAME"
+
+echo "==> Copying $PKG_NAME -> $GH_SHARE_LOCAL/"
+# COPYFILE_DISABLE keeps cp from sprinkling ._* AppleDouble metadata onto
+# the SMB target alongside the package itself.
+COPYFILE_DISABLE=1 cp -p "$PKG" "$DEST_LOCAL"
+
+echo "==> Installing on $UNRAID_HOST"
+echo "    rm -rf /tmp/community.applications /usr/local/emhttp/plugins/community.applications"
+echo "    installpkg $DEST_REMOTE"
+ssh "$UNRAID_HOST" "rm -rf /tmp/community.applications /usr/local/emhttp/plugins/community.applications && installpkg \"$DEST_REMOTE\""
```

Shellcheck (0.11.0)

[info] Note that, unescaped, this expands on the client side.

(SC2029)